### PR TITLE
Support for PHP 8.0

### PR DESCRIPTION
**Pull request by pascalbaljet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,4 +41,4 @@
             "dev-master": "1.1-dev"
         }
     }
-}
+}
```
Comment (tgalopin) [inline composer.json:44]: Could you keep this end of file line?

**Review by tgalopin — COMMENTED** [review]
Thanks, nice efficient PR :) !

File: composer.json (hunk 1 above)
Comment (pascalbaljet) [reply composer.json]: Fixed :)

**Comment by tgalopin** [comment]
Thanks @pascalbaljet !

**Comment by pascalbaljet** [comment]
🎉

**Pull request merged**